src/www/ui/template/admin_content_move.html.twig

```diff
@@ -22,8 +22,8 @@
       <form name="formy" method="post">
         <input type="hidden" name="fromFolder" id="fromFolder" value="-1"/>
 
-        <label for="foldercontent">{{ 'Select the upload or folder you wish to move'|trans }}:<br/>
-          <select name="foldercontent" id="foldercontent" size="20">
+        <label for="foldercontent">{{ 'Select uploads or folders you wish to move'|trans }}:<br/>
+          <select name="foldercontent[]" id="foldercontent" multiple="multiple" size="20">
```

Comment:
so this functionality isn't working as it appears to work?
Multiple select is also possible, but there are no square brackets[]. Is it even sending all selected files?
That's another issue, but maybe a potenial bug. 
![image](https://cloud.githubusercontent.com/assets/6639323/24895968/29800584-1e93-11e7-8ecf-90f7f92ef8de.png)